### ISSUE TITLE
Annotations site documentation page broken on Firefox

### ISSUE THREAD

**Issue by abea** [pattern-lab/patternlab-php#337]
http://patternlab.io/docs/pattern-adding-annotations.html

Latest version of Firefox and FF Developer Edition. Confirmed on multiple machines. I can't tell what's going on other than it seems to have something to do with the `<pre>` element.

**Comment by dap6000** [comment]
Can confirm. Using Firefox 44.0 on Mac OS 10.10.5. Similar behavior in Chrome 48.0.2564.94. Page loads fine in both Opera 34.0 and Safari 9.0.3. 

Seems to be the `<pre>` or maybe the `<code>` element isn't handling the overflow for horizontal scroll properly. But for some reason that is pushing the rest of the page content to the left, off the screen. And I don't get a horizontal scroll bar for the browser window either. I think it is related to flexbox. If I use the Firefox inspector to toggle off `.l-two-col { display: flex; }` then the content fits inside the browser window again. The left hand column is pushed below the main content area, however. If I figure anything else out I'll update here.

**Comment by dap6000** [comment]
Setting a `max-wdith` for  `.content` seemed to clear up the issue in both Firefox and Chrome. The specific values are gonna depend on your break points and what not.

**Comment by patternlab** [comment]
Thanks so much for flagging this. Any way you could share the fix either
through a PR or just through here? I can roll it into the code base.

__________________________________
Brad Frost
http://bradfrost.com
http://twitter.com/brad_frost

On Fri, Jan 29, 2016 at 8:31 AM, Derek Pennycuff <notifications@github.com
<javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:

> Setting a max-wdith for .content seemed to clear up the issue in both
> Firefox and Chrome. The specific values are gonna depend on your break
> points and what not.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pattern-lab/patternlab-php/issues/337#issuecomment-176755419>
> .
>



-- 
__________________________________
Brad Frost
http://bradfrost.com
http://twitter.com/brad_frost

**Comment by dap6000** [comment]
Setting `.content { max-width: 80%; }` passes the Hay test. It doesn't look like shit at any point as I adjust my browser. 

*But* I can't find any references to the `.content` class anywhere in this repo. I can find `.l-two-col` which is the parent of `.content` and I can find `[role=main]` which is the parent of '.l-two-col`. But those contain both the columns in the layout. I can't find a fix to apply there that seems to work. 

I've also tried adjusting attributes like `word-wrap` for the `<code>` and `<pre>` elements but those don't seem to help. The only fix I've discovered in my exploration so far involves setting a rule for a selector I can't find in the repo.

Firefox tells me one of the rules for `.content` is `counter-reset: big-nums;` That seems pretty obscure but I can't find any reference to either the style or the value in this repo either. So I don't seem to be able to even find a mix-in that ends up being applied to `.content`. 

Shit. The website has its own repo. I'm searching the wrong code. I'm switching channels over to https://github.com/pattern-lab/website.
